<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEmployeeDialog</class>
 <widget class="QDialog" name="AddEmployeeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="fullNameLabel">
         <property name="text">
          <string>Полное имя</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="fullNameLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="experienceLabel">
         <property name="text">
          <string>Опыт</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="experienceLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="departmentLabel">
         <property name="text">
          <string>Департамент</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="departmentLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="positionIdLabel">
         <property name="text">
          <string>Должность</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="positionIdLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="addEmployeeButton">
       <property name="text">
        <string>Добавить сотрудника</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
